src/openWelcomeInHubFlag.ts

```diff
@@ -0,0 +1,17 @@
+import { ExtensionContext } from "./preRelease/types";
+
+const ALREADY_OPENED_WELCOME_FLAG = "already-opened-welcome";
+
+export async function setAlreadyInstalledFlag(
```

Comment:
I assume this to avoid opening the welcome/hub again at next restart, where is this called? what triggers this flag?

Reply:
Correct